packages/server/src/api/notifRoutes.test.ts

```diff
@@ -63,3 +64,76 @@ describe("/api/v1/notifications/register/{user_id}", () => {
     });
   });
 });
+
+describe("/api/v1/notifications/phone/register/{user_id}", () => {
+  setupDBTest();
+  const getServer = setupServerTest();
+
+  it("registers a user_i & phone number, tests it's in the db", async () => {
+    const user1 = await UserModel.create({
+      username: "twilioistrello",
+      email: "trello@twilio.com",
+      name: "Big Brain",
+      photoURL:
+        "https://prod-web.neu.edu/wasapp/EnterprisePhotoService/PhotoServlet?vid=CCS&er=471f2d695fbb8a00ee740ad3ea910453986aec81ddaecf889ae98b3a1858597b12650afd0d4e59c561172f76cb1946eec217ed89bd4074c0",
+    }).save();
+
+    const post = await getServer().inject({
+      method: "post",
+      url: `/api/v1/notifications/phone/register/${user1.id}`,
+      payload: {
+        phoneNumber: "+12345678900",
+      },
+    });
+
+    expect(post.statusCode).toBe(200);
+
+    const notifModel = await PhoneNotifModel.findOne();
+    expect(notifModel).toEqual({
+      id: 1,
+      phoneNumber: "+12345678900",
+      user: undefined,
+      userId: 1,
+    });
+  });
+});
+
+describe("/api/v1/notifications/notify_user/{user_id}", () => {
+  setupDBTest();
+  const getServer = setupServerTest();
+
+  it("tests that invalid endpoints for web push get deleted in db", async () => {
```

Comment:
Just curious, where in the code are the endpoints getting deleted?

Reply:
https://github.com/sandboxnu/office-hours/blob/eb4810c226c6c12bf99de5633bb82da393446213/packages/server/src/api/notifRoutes.ts#L135